lib/game.rb

```diff
@@ -0,0 +1,100 @@
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0,1,2], # Top row
+    [3,4,5], # Middle row
+    [6,7,8], # Bottom row
+    [0,3,6], # Left col
+    [1,4,7], # Middle col
+    [2,5,8], # Right col
+    [0,4,8], # Left top to right bottom
+    [2,4,6], # Right top to left bottom
+  ]
+
+  def initialize(player1 = Players::Human.new("X"), player2=Players::Human.new("O"), board = Board.new)
+    @player_1 = player1
+    @player_2 = player2
+    @board = board
+  end
+
+  def current_player
```

Comment:
You can use [`Integer#even?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-even-3F) or [`Integer#odd?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-odd-3F)

`@board.turn_count.even? ? @player_1 : @player_2`